crates/rome_js_syntax/src/declaration_ext.rs

```diff
@@ -9,6 +9,13 @@ impl TsEnumDeclaration {
     }
 }
 
+impl JsClassDeclaration {
+    /// Returns `true` if this enum is an ambient enum or in an ambient context.
```

Comment:
The documention isn't correct, because this function is associated with a class declaration, not an enum.